EBA/Taschenrechner/MatheFunktionen/Mathe.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Taschenrechner
+{
+    public partial class MatheForm : Form
+    {
+        public double helper = 0;
```

Comment:
this blog is the same like in `GeometrieForm`. Maybe you want create a base class for this?

Reply:
maybe 🤷‍♂️ but can we assume that my teacher will understand this if i create a class, from which almost all classes inherit therefrom

Reply:
Yes this is the basic stuff of oop 🤷‍♀️

Reply:
if you say so :)

Reply:
Done :D 
 I added an Base Form Class from which all other moduls inherit

Reply:
@paule96 any suggestions on how to improve this even further?